packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -38,10 +57,28 @@ const CertificationQuiz: React.FunctionComponent = () => {
     }
   }
 
+  const { mutate: postCertifiedUserTestResponse, isLoading: isSubmitting } =
+    usePostCertifiedUserTestResponse({
+      onSuccess: () => {
+        window.scrollTo(0, 0)
+      },
+      onError: e => {
+        handleError(e)
+      },
+    })
+  // user is taking the quiz if user is not certified, and either there is no passing record or if the user clicked to retake the quiz
+  const isTakingQuiz =
+    !isCertified && (passingRecord == undefined || isRetakingQuiz)
   useEffect(() => {
-    getQuiz()
+    if (accessToken) {
+      getQuiz()
```

Comment:
Continue to always get a fresh Quiz when component is mounted.  But it's not always used.